WFP-VAM/prism-app
# Add layer config access in legend

This PR is still drafted to fixes issue #793 

How to test the feature:

- activate any layer
- check the icons on the legend
